feat: ability to run test recursively (#25)
VeryGoodOpenSource/very_good_workflows: README.md

**Default** `"."`

#### `test_recursion`

> **Optional** If it should run the test recursively.

Reviewer: ```suggestion
**Optional** Whether to recursively run tests in nested directories.
```
